.eslintrc.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
     // Disabled checks
     'class-methods-use-this': 'off',
     'import/prefer-default-export': 'off',
+    'react/jsx-props-no-spreading': 'off',
+    'react/require-default-props': 'off',
```

Comment:
` 'react/jsx-props-no-spreading': 'off',` -> props spreading can be useful in some context, you can't stop me.
`'react/require-default-props': 'off',` -> required default props syntax is more exhaustive than useful, and this rule is usually not included in recommended react rules set.